docs/src/pages/docs/functions/helpers.md

```diff
@@ -340,6 +340,59 @@ you provide an `undefined` values for the second, that `Pair` will not be
 represented in the resulting `Object`. Also, when if multiple keys share the
 same name, that last value will be moved over.
 
+#### getPathOr
+
+`crocks/helpers/getPathOr`
+
+```haskell
+getPathOr :: a -> [ (String | Integer) ] -> b -> c
+```
+
+While [`getPropOr`](#getpropor) is good for simple, single-level
+structures, there may come a time when you have to work with nested POJOs or
+Arrays. When you run into this situation, just pull in `getPathOr`, which was
+previously called `propPathOr`, and pass it a left-to-right traversal path of
+keys, indices or a combination of both (gross...but possible). This will kick
+you back a function that behaves just like [`getPropOr`](#getpropor). You pass
+it some data, and it will attempt to resolve your provided path. If the path is
+valid, it will return the value. But if at any point that path "breaks" it will
+give you back the default value.
+
+```javascript
+import getPathOr from 'crocks/helpers/getPathOr'
+
+const data = {
+  foo: {
+    bar: 'bar',
+    null: null,
+    nan: NaN,
+    undef: undefined
+  },
+  arr: [ 1, 2 ]
+}
+
+// def :: [ String | Integer ] -> a -> b
+const def =
+  getPathOr('default')
+
+def([ 'foo', 'bar' ], data)
+//=> "bar"
+
+def([ 'baz', 'tommy' ], data)
+//=> "default"
+
+def([ 'foo', 'null' ], data)
+//=> null
+
+def([ 'foo', 'nan' ], data)
+//=> NaN
+
+def([ 'foo', 'undef' ], data)
+//=> "default"
+
+def([ 'arr', 'length' ], data)
+//=> 2
+```
 
```

Comment:
The is no additional changes to the `#### propPathOr` section, would we want to mention there that this is now deprecated and reference the new name?

Reply:
Also in that example, would we want to switch it to using the new name as well? These are just curiosity questions

Reply:
Ohhh great catches. Rebase issue with a stale branch.
Example will be switched and `propPathOr` will be removed from page entirely.
Will update

Reply:
Section removed.
Great Catch!